mypy/test/testsolve.py

```diff
@@ -2,16 +2,15 @@
 
 from typing import List, Union, Tuple, Optional
 
-from mypy.myunit import Suite, assert_equal
+from mypy.test.helpers import Suite, assert_equal
 from mypy.constraints import SUPERTYPE_OF, SUBTYPE_OF, Constraint
 from mypy.solve import solve_constraints
 from mypy.test.typefixture import TypeFixture
 from mypy.types import Type, TypeVarType, TypeVarId
 
 
 class SolveSuite(Suite):
-    def __init__(self) -> None:
-        super().__init__()
+    def setUp(self) -> None:
```

Comment:
Elsewhere we had a `setup` method, which is a bit confusing. This is needed because of `unittest`. I wonder if it make sense to use this name variant uniformly?

Reply:
If you insist, but I'm not sure that's desirable. The naming distinction makes it clear where it's required by unittest and where it isn't.

Reply:
Ok no need to change this. If this becomes a problem we can reconsider.